openvasreporting/libs/parser.py

```diff
@@ -10,10 +10,10 @@
 from .config import Config
 from .parsed_data import Host, Port, Vulnerability
 
-# logging.basicConfig(stream=sys.stderr, level=logging.DEBUG,
-#                     format="%(asctime)s | %(levelname)s | %(name)s | %(message)s", datefmt="%Y-%m-%d %H:%M:%S")
-logging.basicConfig(stream=sys.stderr, level=logging.ERROR,
-                    format="%(asctime)s | %(levelname)s | %(name)s | %(message)s", datefmt="%Y-%m-%d %H:%M:%S")
+logging.basicConfig(stream=sys.stderr, level=logging.DEBUG,
+                     format="%(asctime)s | %(levelname)s | %(name)s | %(message)s", datefmt="%Y-%m-%d %H:%M:%S")
+#logging.basicConfig(stream=sys.stderr, level=logging.ERROR,
+#                    format="%(asctime)s | %(levelname)s | %(name)s | %(message)s", datefmt="%Y-%m-%d %H:%M:%S")
```

Comment:
Don't forget to revert this to prevent unnecessary changes

Reply:
agred